### PR TITLE
Update debian installation to use python3

### PR DESCRIPTION
**Pull request by danielguardicore**
# What is this? 
Change python2 to Python3 in a debian installation package.

## Checklist
* [X] Have you added an explanation of what your changes do and why you'd like to include them?
* [X] Have you successfully tested your changes locally?
* [ ] Is the TravisCI build passing?

### CODE REVIEW

File: monkey/monkey_island/deb-package/DEBIAN/control [hunk 1]
```diff
@@ -5,4 +5,4 @@ Homepage: http://www.guardicore.com
 Priority: optional
 Version: 1.0
 Description: Guardicore Infection Monkey Island installation package
-Depends: openssl, python-pip, python-dev
+Depends: openssl, python3-pip, python3-dev
```
Comment (VakarisZ) [inline monkey/monkey_island/deb-package/DEBIAN/control]: does this guarantee 3.7?

File: monkey/monkey_island/deb-package/DEBIAN_MONGO/postinst [hunk 2]
```diff
@@ -5,8 +5,8 @@ INSTALLATION_FOLDER=/var/monkey/monkey_island/installation
 PYTHON_FOLDER=/var/monkey/monkey_island/bin/python
 
 # Prepare python virtualenv
-pip2 install virtualenv --no-index --find-links file://$INSTALLATION_FOLDER
-virtualenv -p python2.7 ${PYTHON_FOLDER}
+pip3 install virtualenv --no-index --find-links file://$INSTALLATION_FOLDER
+virtualenv -p python3 ${PYTHON_FOLDER}
```
Comment (VakarisZ) [inline monkey/monkey_island/deb-package/DEBIAN_MONGO/postinst]: also, should be 3.7, no?

File: monkey/monkey_island/deb-package/DEBIAN/control (hunk 1 above)
Comment (danielguardicore) [reply monkey/monkey_island/deb-package/DEBIAN/control]: No. It guarantees pip compatible with py3 and py3 headers for pypi package compilation.

File: monkey/monkey_island/deb-package/DEBIAN_MONGO/postinst (hunk 2 above)
Comment (danielguardicore) [reply monkey/monkey_island/deb-package/DEBIAN_MONGO/postinst]: Then we break on Py3.8 and Py3.9, along with Py3.6 where we currently still work

**Pull request merged**